src/app/core/user-module/user.service.ts

```diff
@@ -79,16 +51,17 @@ export class UserService {
    * @returns {Promise<User>}
    * @memberof UserService
    */
-  editMember(collection: string, memberId: string, role: string): Promise<User> {
+  editMember(
+    collection: string,
+    memberId: string,
+    role: string
+  ): Promise<User> {
     return this.http
-      .patch(USER_ROUTES.UPDATE_COLLECTION_MEMBER(collection, memberId), role, {
+      .patch(LEGACY_USER_ROUTES.UPDATE_COLLECTION_MEMBER(collection, memberId), role, {
```

Comment:
Permissions should be in access group module and use the current routes that are implemented